app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+	before_save {self.email = email.downcase}
```

Comment:
```suggestion
	before_save {email.downcase!}
```
so sánh giúp a .downcase vs .downcase! nhé

Reply:
dạ vâng em cám ơn a

Reply:
> so sánh giúp a .downcase vs .downcase! nhé

chưa trả lời này e